code/datums/viewport.dm

```diff
@@ -0,0 +1,100 @@
+#define ceil(x) (-round(-(x)))
+/client
```

Comment:
I'm not going to go through this entire file with this feedback, but we are not Skyrat and we do not split typepaths like this. As much as I'd like to (it's great for surface area), It causes problems for the LSP that we do not have solutions too yet, which make the majority use case hard to use.

Reply:
I don't know what you mean by "split typepaths"

Reply:
what they mean is that, new client vars should all be grouped into the same place (`code/modules/client/client_defines.dm`)

Reply:
done, found the ROUND_UP macro does just this so I don't need a new macro